docs/SENTENCES.md

```diff
@@ -34,8 +41,21 @@ Feel free to set up this QA however makes most sense for you, but here's a [samp
 
 Once the review is complete, submit a pull request with the # of sentences submitted, a link to the manual QA results, and the % error rate. Here's [an example PR](https://github.com/mozilla/common-voice/pull/2873). Please make sure the sentences are in a plain `.txt` file with one sentence per line.
 
-## Flagging problematic sentences
+## Rules for QA that applies (or not) to the different imputs and outputs
+To help you, for exemple with the 'not supervised, do your own QA' bulk submission, you _may_ find interesting to have a look at : 
+* [cleanup](https://github.com/common-voice/sentence-collector/tree/main/server/lib/cleanup) and [validation](https://github.com/common-voice/sentence-collector/tree/main/server/lib/validation) rules for sentence collector, and/or 
+* [language](https://github.com/common-voice/cv-sentence-extractor#using-language-rules) rules for extractor, or even 
+* [preprocessor](https://github.com/common-voice/CorporaCreator/tree/master/src/corporacreator/preprocessors) for CorporaCreator ([CorporaCreator](https://github.com/common-voice/CorporaCreator) is the Common Voice extractor that, linking text and recorded voice samples, make them available for [SST](https://en.wikipedia.org/wiki/Speech_recognition) engines' training).
+
+ 
+# Correcting existing data
+
+As you can see, some methods doesn't go through automated cleanup/validation/rules, and they are not unified ([...not yet](https://discourse.mozilla.org/t/sentence-collector-cleanup-before-export-vs-cleanup-on-upload/105411/15)). Thus, there is a process to remove old data that might need to be discarded.
```

Comment:
```suggestion
Some methods don't go through automated cleanup/validation/rules, and they are not unified ([...not yet](https://discourse.mozilla.org/t/sentence-collector-cleanup-before-export-vs-cleanup-on-upload/105411/15)). Thus, there is a process to remove old data that might need to be discarded.
```

.. additionally, I do not think this is the right link to link. If anything, this should point to a dedicated discussion thread around this topic, or a GitHub issue. Linking just that comment does not provide more info that "not yet" states.

Additionally, this is not the only reason why a sentence would be removed. I would like to keep this more general here.

Reply:
I don't want to start a fire that I won't be able to handle, but I don't understand why cleanup and validation are not unified. ...Is it by choice / design, or only by "legacy" ? 

Shall we open an issue saying "We should unify cleanup and validation between sentence collector, sentence extractor and corpora creator" ?

Reply:
> I don't want to start a fire that I won't be able to handle, but I don't understand why cleanup and validation are not unified. ...Is it by choice / design, or only by "legacy" ?

Out of those choices, I'd say legacy applies best here. In the end these tools also use different technologies, however that would be solve-able in the end. There are also slight differences in purposes, however that could probably also be dealt with, though I have to admit that I don't know the exact workings of the CorporaCreator.

Yes, something to track a possible "one rule file fits all" solution would be great, and that, in my opinion, could then also be linked here. As this spans multiple repos, maybe we should start with a [Discourse](https://discourse.mozilla.org/c/voice/239) post, and then go from there and once we agree on something create issues in the appropriate repositories.

Reply:
I'm starting with "let's have an overview", then I'll follow by "now we can see that we do everywhere some cleanup and validation, shall we have a unified verification and cleanup preprocessor, as for all inputs, and before output/corpora creator"

I think with solve this conversation...

https://discourse.mozilla.org/t/trying-to-create-a-common-voice-overview-for-newbies-svg-drawin-style/105771